packages/ember-concurrency/babel.config.json

```diff
@@ -0,0 +1,12 @@
+// Babel config for building the ember-concurrency package itself. Because EC is a V2 addon,
+// this babel config should *not* apply presets or compile away ES modules.
+// It exists only to provide development niceties for you, like automatic
+// template colocation.
+{
+  "plugins": [
+    // https://github.com/embroider-build/embroider/issues/1014
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    "@babel/plugin-proposal-class-properties",
```

Comment:
Any reason not to use the current babel config from the blueprint?

Reply:
I updated to the latest blueprint, thank you!